src/jit/gentree.h

```diff
@@ -1032,6 +1033,7 @@ struct GenTree
 
 #define GTF_ARR_BOUND_INBND         0x80000000 // GT_ARR_BOUNDS_CHECK -- have proved this check is always in-bounds
 
+#define GTF_ARRLEN_NONFAULTING      0x20000000 // GT_ARR_LENGTH  -- An array length operation that cannot fault. Same as GT_IND_NONFAULTING.
```

Comment:
Mind moving this down a line to be below `GTF_ARRLEN_ARR_IDX`?  The flags in this file are typically listed in descending numerical order within each group.

Reply:
Done.

Reply:
Perfect, thanks.